Types.lua

```diff
@@ -57,6 +57,9 @@ WeakAuras.trigger_require_types = {
   all = L["All Triggers"],
   custom = L["Custom Function"]
 };
+WeakAuras.trigger_modes = {
+  ["first_active"] = -10,
+};
 WeakAuras.trigger_types = {
```

Comment:
More are certainly possible, I was considering: Show the x auras with the least remaining time, Show x auras with highest stacks.